nuget/framework/nunit.nuspec

```diff
@@ -50,7 +49,11 @@ Supported platforms:
     <file src="bin/net-4.5/nunit.framework.xml" target="lib/net45" />
     <file src="bin/netstandard16/nunit.framework.dll" target="lib/netstandard1.6" />
     <file src="bin/netstandard16/nunit.framework.xml" target="lib/netstandard1.6" />
-    <file src="bin/portable/nunit.framework.dll" target="lib/portable-net45+win8+wp8+wpa81+Xamarin.Mac+MonoAndroid10+MonoTouch10+Xamarin.iOS10" />
-    <file src="bin/portable/nunit.framework.xml" target="lib/portable-net45+win8+wp8+wpa81+Xamarin.Mac+MonoAndroid10+MonoTouch10+Xamarin.iOS10" />
+    <file src="bin/portable/nunit.framework.dll" target="lib/portable-net45+win8+wp8+wpa81" />
+    <file src="bin/portable/nunit.framework.xml" target="lib/portable-net45+win8+wp8+wpa81" />
+    <file src="bin/portable/nunit.framework.dll" target="lib/MonoAndroid" />
+    <file src="bin/portable/nunit.framework.xml" target="lib/MonoAndroid" />
+    <file src="bin/portable/nunit.framework.dll" target="lib/Xamarin.iOS10" />
+    <file src="bin/portable/nunit.framework.xml" target="lib/Xamarin.iOS10" />
```

Comment:
We don't have any runners for Xamarin.Mac and MonoTouch is now Xamarin.iOS10 AFAIK.

As for NetStandard 1.4, our PCL is currently at least 1.3 if not 1.0 compliant, so I think we should drop PCL for the next release and switch it to .NET Standard 1.0 or 1.3. Checking other common packages, they have already done so.